src/Libraries/RevitNodes/Elements/Views/StructuralPlanView.cs

```diff
@@ -72,6 +73,8 @@ public static StructuralPlanView ByLevel(Level level)
         {
             if (level == null)
             {
+                LogWarningMessageEvents.OnLogInfoMessage("StructuralPlans can only be created when the structural discipline is enabled whereas FloorPlans and CeilingPlans can only be created when architecture or MEP disciplines are enabled. " +
+                    "Check which tabs are active under File -> Options -> User Interface -> Configure Tools and analysis");
                 throw new ArgumentNullException("level");
```

Comment:
Same comment here about making this a warning since you're throwing an exception, or maybe including the warning message in the argument to the exception itself. That way, it should automatically appear in the node warning.